lbrynet/core/LBRYWallet.py

```diff
@@ -78,13 +83,59 @@ def __init__(self, db_dir):
         self._batch_count = 20
         self._first_run = self._FIRST_RUN_UNKNOWN
 
+        self._USDBTC = None
```

Comment:
I'm not convinced mixing the currency conversion logic inside of the wallet is the right path.  The wallet acts as an abstraction over lbrycrd/lbryum and is responsible for interacting with the bitcoin network.

In SOLID design (https://en.wikipedia.org/wiki/SOLID_(object-oriented_design)) classes should have a single responsibility.  

I would suggest making a new class, maybe `CurrentExchangeRates`.  I would also make another class, `ExchangeRate`.  The later would be an abstraction for the the functions calls that you have (USD->BTC, BTC->LBC).  The former would act as a container for the `ExchangeRates` and be responsible for ensuring the freshness of the quotes.